### DISCUSSION TITLE
Ability to pull AHS reports from 1 or multiple computes.

### DISCUSSION THREAD

**Discussion by JaragonCR** [HewlettPackard/POSH-HPOneView#551] [Q&A] [answered]
This is an enhancement request

Would it be possible to create a command to pull AHS reports from a select set of servers or even "stagger" all servers?

**Comment by ChrisLynchHPE** [comment, marked as answer]
HPE OneView can initiate an AHS collection, but only for when OneView Remote Support is enabled.  This can be controlled with the [Start-OVRemoteSupportDataCollection](https://hpe-docs.gitbook.io/posh-hpeoneview/cmdlets/v5.50/appliance/start-ovremotesupportcollection) Cmdlet.
